src/main/webapp/profile_creation.jsp

```diff
@@ -5,7 +5,10 @@
 <t:genericpage>
     <jsp:body>
        <div class="content">
-        <form action="/profile_edit" method="POST">
+        <h3> User Registration </h3>
```

Comment:
I think you should remove the space between the tags and the text,
`        <h3> User Registration </h3>
`
would be
`        <h3>User Registration</h3>
`.